docs/source/features/metrics.md

```diff
@@ -34,6 +34,43 @@ The API key can also be set with the `ENGINE_API_KEY` environment variable. Sett
 ENGINE_API_KEY=YOUR_API_KEY node start-server.js
 ```
 
+### Client Awareness
+
+Apollo Engine accepts metrics annotated with client information. The Engine UI
+is then able to filter metrics and usage patterns by these names and versions. To provide metrics to the Engine, pass a `generateClientInfo` function into the `ApolloServer` constructor, like so:
```

Comment:
I think this should also reference the default implementation? Something like "by default, Apollo Server uses a function that looks in "query extensions" for client information and forwards them on if present. You can override that function by specifying `generateClientInfo` yourself with a link to the default generator or file enclosing it and a reasonable override (or c/p of code)

Reply:
@evans do we want to change this section to mention the default / provide an implementation that's different from the default (e.g. headers)?